src/components/views/read/cvForm/types.d.ts

```diff
@@ -0,0 +1,67 @@
+declare type PersonalInfoObj = {
```

Comment:
This creates an `ambiant type` this is not what you want. 

```suggestion
export type PersonalInfoObj = {
```